api/client/formatter/service.go

```diff
@@ -0,0 +1,288 @@
+package formatter
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/client/inspect"
+	"github.com/docker/engine-api/types/swarm"
+	units "github.com/docker/go-units"
+)
+
+const serviceInspectPrettyTemplate = `
```

Comment:
we should add some examples to https://docs.docker.com/engine/admin/formatting/

(I also didn't know of `{{- }}`)